jupyter_sigplot/sigplot.py

```diff
@@ -217,6 +188,42 @@ def overlay_file(self, path):
         self.overlay_href(path)
 
 
+
+def _prepare_array_input(input, layer_type, subsize):
+    # TODO (sat 2018-11-08): I think this needs to move into a
+    # function so we can test it better. At that point, we may
+    # also be able to specify it as the serializer for the
+    # traitlet via the `to_json` keyword argument
+    # TODO (ddw 2018-12-10): Do I need to check the number of
+    # items in input?
+    data, header = input
```

Comment:
You shouldn't need to check the number of items in input as long as you ensure `input` is always a tuple consisting of `(data, header)` (with the type `Tuple[List, Dict]`).